# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/1825450.html
# Commit: a83043c7ab51
# Full Hash: a83043c7ab51233d38f3f65704de87aa3db3feac
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2023-07-10 09:40:14
# Description:
#   Bug 1825450 - Ensure device pixel scale for surfaces can't become infinite r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D182429
# ==============================================================================

diff -r d3ca9e70625b -r a83043c7ab51 gfx/tests/crashtests/1825450.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/tests/crashtests/1825450.html	Sun Jul 09 21:22:10 2023 +0000
@@ -0,0 +1,15 @@
+<style>
+*:last-child {
+  transform: skew(360deg, 90deg);
+  border-left: 400px inset rgba(212, 81, 240, 139);
+  scale: 1 65536;
+  clip-path: inset(100% round 71px 0% / 100% 87%) border-box
+}
+</style>
+<script>
+window.onload = () => { document.dir = "rtl" }
+</script>
+<button>
+<h1>
+<input>
+